jest.config.js

```diff
@@ -8,4 +8,11 @@ module.exports = {
     // Needed to use @shelf/jest-mongodb preset.
     ...tsJestPreset.transform,
   },
+  collectCoverageFrom: ['./src/**/*.{ts,js}'],
```

Comment:
Should we only collect coverage for `ts` files, since jest only works for ts files currently?